contrib/packs/actions/workflows/create.yaml

```diff
@@ -0,0 +1,51 @@
+---
+vars:
+  pack_path: '/opt/stackstorm/packs/{{ name }}'
+  pack_group: stanley
+  pack_permissions: '775'
+
+chain:
+  -
+    name: "create directory"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "mkdir -p {{ pack_path }}"
+    on-success: "change permissions"
+  -
+    name: "change permissions"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "chmod -R {{ pack_permissions }} {{ pack_path }}"
+    on-success: "change ownership"
+  -
+    name: "change ownership"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "chgrp -R {{ pack_group }} {{ pack_path }}"
+    on-success: "render pack.yml"
+  -
+    name: "render pack.yml"
```

Comment:
We should probably make it `.yaml` and not `.yml`, for consistency's sake. All the packs we have use `pack.yaml`, and even if we rename meta in every pack to `.yml` (for whatever reason), everything else in StackStorm has the `.yaml` extension, too.

So if there's no strong point for it, I'd just change the default file to `.yaml`. `yml` is nice, but more trouble than it's worth.